ArduCopter/flight_mode.cpp

```diff
@@ -142,17 +142,17 @@ bool Copter::set_mode(control_mode_t mode, mode_reason_t reason)
     control_mode_reason = reason;
     DataFlash.Log_Write_Mode(control_mode);
 
-        adsb.set_is_auto_mode((mode == AUTO) || (mode == RTL) || (mode == GUIDED));
+    adsb.set_is_auto_mode((mode == AUTO) || (mode == RTL) || (mode == GUIDED));
 
 #if AC_FENCE == ENABLED
-        // pilot requested flight mode change during a fence breach indicates pilot is attempting to manually recover
-        // this flight mode change could be automatic (i.e. fence, battery, GPS or GCS failsafe)
-        // but it should be harmless to disable the fence temporarily in these situations as well
+    // pilot requested flight mode change during a fence breach indicates pilot is attempting to manually recover
+    // this flight mode change could be automatic (i.e. fence, battery, GPS or GCS failsafe)
+    // but it should be harmless to disable the fence temporarily in these situations as well
     fence.manual_recovery_start();
 #endif
         
 #if FRSKY_TELEM_ENABLED == ENABLED
-        frsky_telemetry.update_control_mode(control_mode);
+    frsky_telemetry.update_control_mode(control_mode);
 #endif
 #if CAMERA == ENABLED
         camera.set_is_auto_mode(control_mode == AUTO);
```

Comment:
This one also has the indentation wrong.

Reply:
txs, fixed in follow-up commit.